input/resources/OperationDefinition-evaluate-measure.xml

```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<OperationDefinition
+    xmlns="http://hl7.org/fhir">
+    <id value="evaluate-measure"/>
+    <url value="http://hl7.org/fhir/us/davinci-deqm/OperationDefinition/evaluate-measure"/>
+    <name value="EvaluateMeasure"/>
+    <title value="EvaluateMeasure"/>
+    <status value="active"/>
+    <kind value="operation"/>
+    <publisher value="Clinical Quality Information Workgroup"/>
+    <contact>
+      <telecom>
+        <system value="url"/>
+        <value value="http://www.hl7.org/Special/committees/cqi/index.cfm"/>
+      </telecom>
+    </contact>
+    <description value="The evaluate-measure operation is used to calculate an eMeasure and obtain the results."/>
+    <jurisdiction>
+      <coding>
+        <system value="urn:iso:std:iso:3166"/>
+        <code value="US"/>
+        <display value="United States of America"/>
+      </coding>
+    </jurisdiction>
+    <code value="evaluate-measure"/>
+    <base value="http://hl7.org/fhir/OperationDefinition/Measure-evaluate-measure"/>
```

Comment:
If .base refers to R4, and this DEQM opdef is like a "profile" of it, then a profile cannot relax a constraint.  The R4 base "return" out param is 1..1, and we cannot relax if to 0..1 or 0..* (if R4 says it is mandatory, we cannot make it optional.  We can if we change the .code to use a different name; i.e., it is essentially a different operation.  Changing the type from MeasureReport to Bundle would also not be allowed.  We could add a 2nd out param that is Bundle 0..*, but that does not get us away from return of a MeasureReport being 1..1 (Mandatory).

Reply:
See https://chat.fhir.org/#narrow/stream/179252-IG-creation/topic/Profiling.20an.20operation and https://chat.fhir.org/#narrow/stream/179252-IG-creation/topic/R4.20and.20R5.20implementation.20of.20the.20same.20operation for more discussion

Reply:
We can discuss this. I thought about naming the evaluate-measure here as deqm.evaluate-measure following the pattern we used in risk adjustment to prefix the operation.